spec.html

```diff
@@ -11976,6 +12002,20 @@ <h1>
         <p>The _realm_ for Jobs returned by NewPromiseResolveThenableJob is usually the result of calling GetFunctionRealm on the _then_ function object. The _realm_ for Jobs returned by NewPromiseReactionJob is usually the result of calling GetFunctionRealm on the handler if the handler is not *undefined*. If the handler is *undefined*, _realm_ is *null*. For both kinds of Jobs, when GetFunctionRealm completes abnormally (i.e. called on a revoked Proxy), _realm_ is the current Realm at the time of the GetFunctionRealm call. When the _realm_ is *null*, no user ECMAScript code will be evaluated and no new ECMAScript objects (e.g. Error objects) will be created. The WHATWG HTML specification (<a href="https://html.spec.whatwg.org/">https://html.spec.whatwg.org/</a>), for example, uses _realm_ to check for the ability to run script and for the <a href="https://html.spec.whatwg.org/#entry">entry</a> concept.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-hostenqueuetimeoutjob" type="host-defined abstract operation">
+      <h1>
+        HostEnqueueTimeoutJob (
+          _timeoutJob_: a Job Abstract Closure,
+          _milliseconds_: a non-negative finite Number,
+        ): ~unused~
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It schedules _timeoutJob_ to be performed after at least _milliseconds_ milliseconds.</dd>
+      </dl>
+      <p>An implementation of HostEnqueueTimeoutJob must conform to the requirements in <emu-xref href="#sec-jobs"></emu-xref>.</p>
```

Comment:
Hm, actually, is this not required to run in any particular agent? It's up to the host?

If that's so my suggestion above doesn't make sense, but we still should probably reword it to make it clearer that the agent to run the job in is sometimes specified.